test/Microsoft.TemplateEngine.Orchestrator.RunnableProjects.UnitTests/TemplateConfigTests/TestTemplateSetup.cs

```diff
@@ -60,12 +59,15 @@ private IMountPoint SourceMountPoint
             {
                 if (_sourceMountPoint == null)
                 {
-                    if (!new FileSystemMountPointFactory().TryMount(_environment, null, _sourceBaseDir, out IMountPoint sourceMount))
+                    foreach (var factory in _environmentSettings.SettingsLoader.Components.OfType<IMountPointFactory>())
                     {
-                        Assert.True(false, "couldn't create source mount point");
+                        if (factory.TryMount(_environmentSettings, null, _sourceBaseDir, out var myMountPoint))
```

Comment:
using the concrete type instead of `var` may add more clarity in terms of maintainability.